backend/models/professor-services.js

```diff
@@ -82,6 +82,25 @@ async function professorRatingUpdate(prof, newRating) {
   return newNum, newAvg;
 }
 
+// get all reviews for a professor - return null if professor does not exist
+async function getAllReviews(professor) {
+  try {
+    const professorModel = Connection.getConnection().model(
+      "Professor",
+      ProfessorSchema
+    );
+    const professorToFind = await professorModel.findOne({ name: professor });
+    if (professorToFind === null) {
+      return professorToFind;
```

Comment:
is there a test case that expects this return statement? (aka testing the null case)

Reply:
great catch!  I will add one.